pkg/gds/interceptor.go

```diff
@@ -36,7 +64,14 @@ func (s *Service) serverInterceptor(ctx context.Context, in interface{}, info *g
 	}
 
 	// Call the handler to finalize the request and get the response.
+	var span *sentry.Span
+	if trackPerfomance {
+		span = sentry.StartSpan(ctx, info.FullMethod)
```

Comment:
How does Sentry differentiate between the members DSN and the API DSN here?

Reply:
I don't think it actually does, at least the way Sentry is configured in the current state of this PR, so I think that's another reason to combine them for now.